pdf_to_csv.py

```diff
@@ -156,13 +180,47 @@ def generate_page_table_data(self, input_pdf_filepath, input_pdf_obj,
             logger.warning(warning_message.format(page_num, input_pdf_filepath))
         return page_table_data
 
+    def log_data(self, input_pdf_filepath, page_num, measure_file_obj,
```

Comment:
log_monitoring_data or log_performance_data, if possible let's keep vocab consistent throughout the file?